core-bundle/config/listener.yaml

```diff
@@ -69,6 +69,14 @@ services:
             - '@security.helper'
             - '@request_stack'
 
+    contao.listener.data_container.content.description_list_label:
+        class: Contao\CoreBundle\EventListener\DataContainer\Content\DescriptionListLabelListener
+        public: true
+        arguments:
+            - '@translator'
+        tags:
+            - { name: contao.callback, table: tl_content, target: fields.data.attributes }
```

Comment:
It could be, yes. This was probably added before we had `autoconfigure: true`. There are other listeners here though with tags. Technically it does not make a difference, I would suggest to review all tags-instead-of-attributes in a separate PR.